bin/hwinj/pycbc_generate_hwinj

```diff
@@ -141,6 +141,10 @@ parser.add_argument('--mass2', type=float, required=True,
                     help='Second mass of the binary in solar masses.')
 parser.add_argument('--inclination', type=float, required=True,
                     help='Inclination of the binary in radians.')
+parser.add_argument('--coa-phase', type=float, default=0.0,
+                    help='Reference orbital phase parameter in radians. '
```

Comment:
Could you point me to where I can find the exact definition of the parameters in the sim inspiral table? I couldn't find it.

But if it corresponds to the phiRef parameter in lalsimulation then it is the reference *orbital* phase (at the reference GW frequency)

See: https://github.com/lscsoft/lalsuite/blob/master/lalsimulation/src/LALSimInspiral.c#L311